server/server.js

```diff
@@ -0,0 +1,6 @@
+const PORT = process.env.PORT || 4000;
```

Comment:
I suggest naming this file to index (default entry point in nodejs)

Reply:
I did it @FarahZaqout